internal/ingress/metric/main.go

```diff
@@ -175,6 +177,14 @@ func (c *collector) SetSSLExpireTime(servers []*ingress.Server) {
 	c.ingressController.SetSSLExpireTime(servers)
 }
 
+func (c *collector) IncOrphanIngress(namespace string, name string, type_ string) {
```

Comment:
why this variable name with underline?

Reply:
https://github.com/kubernetes/ingress-nginx/pull/8230/files#r805408344

Reply:
don't get it @alex123012

Reply:
sry @alex123012 did not read everything before commenting ^^